packages/brokeneck-fastify/lib/plugins/auth/azure/operations.js

```diff
@@ -46,7 +46,48 @@ const listNextOperationSpec = {
   serializer
 }
 
+const listGroupsOperationSpec = {
+  httpMethod: 'GET',
+  path: '{tenantID}/groups',
+  urlParameters: [Parameters.tenantID],
+  queryParameters: [
+    Parameters.filter,
+    Parameters.top,
+    Parameters.apiVersion,
+    Parameters.search
+  ],
+  headerParameters: [Parameters.acceptLanguage],
+  responses: {
+    200: {
+      bodyMapper: Mappers.GroupListResult
+    },
+    default: {
+      bodyMapper: Mappers.GraphError
+    }
+  },
+  serializer
+}
```

Comment:
you may have figured that I had to do this because the SDK doesn't support all parameters, so I had to switch to lower-level operations in order to be able to provide those parameters, as the HTTP does support them but the client doesn't. I think we need to consider switching to invoking the HTTP API directly instead of having to reverse engineer the SDK.